services/venice-controller/src/test/java/com/linkedin/venice/controller/TestDisabledPartitionEnablerService.java

```diff
@@ -0,0 +1,40 @@
+package com.linkedin.venice.controller;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.atLeast;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import com.linkedin.venice.utils.TestMockTime;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import org.testng.annotations.Test;
+
+
+public class TestDisabledPartitionEnablerService {
+  @Test
+  public void testCleanupBackupVersion() throws Exception {
+    VeniceHelixAdmin admin = mock(VeniceHelixAdmin.class);
+    String clusterName = "test_cluster";
+    VeniceControllerMultiClusterConfig config = mock(VeniceControllerMultiClusterConfig.class);
+    long defaultRetentionMs = TimeUnit.DAYS.toMillis(7);
+    doReturn(defaultRetentionMs).when(config).getBackupVersionDefaultRetentionMs();
+    VeniceControllerConfig controllerConfig = mock(VeniceControllerConfig.class);
+    doReturn(controllerConfig).when(config).getControllerConfig(anyString());
+    Set<String> clusters = new HashSet<>();
+    doReturn(true).when(admin).isLeaderControllerFor(any());
+    clusters.add(clusterName);
+    doReturn(true).when(controllerConfig).isEnableDisabledReplicaEnabled();
+    doReturn(clusters).when(config).getClusters();
+    TestMockTime time = new TestMockTime();
+    DisabledPartitionEnablerService service = new DisabledPartitionEnablerService(admin, config, time);
+
+    service.startInner();
+    verify(admin, atLeast(1)).enableDisabledPartition(clusterName, "", true);
```

Comment:
This checks whether `enableDisabledPartition()` is being called in the timer, which is great, but should we also add a test to disable multiple partitions and check if it's re-enabled by this service?